ols/src/query_helpers/__init__.py

```diff
@@ -1 +1,28 @@
 """Question validators, statament classifiers, and response generators."""
+import re
+
+from ols.utils import config
+from ols.utils.logger import Logger
+
+
+def camel_to_snake(string: str) -> str:
+    """Convert a string from camel case to snake case.
+
+    Args:
+        string: The string to be converted.
+
+    Returns:
+        The converted string.
+    """
+    string = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", string)
+    return re.sub("([a-z0-9])([A-Z])", r"\1_\2", string).lower()
+
+
+class QueryHelper:
```

Comment:
We also have `DocsSummarizer`. It is currently loading configuration from env. I would propose to move it into `query_helpers/` and make it a subclass of  `QueryHelper` too. WDYT?

Reply:
probably worth to put into utils if needed in other places?

Reply:
What part? This `QueryHelper` parent?